omod-common/src/test/java/org/mockingbird/test/rest/resource/DuplicateNameAnimalResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
Are these new methods also fully documented anywhere in the code? Am looking at it from the point of the maintainers who will look at these new methods and try to figure out why they were added.